.github/workflows/dco.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+# yamllint disable rule:line-length
+
+name: "DCO check"
+
+# yamllint disable-line rule:truthy
+on: [pull_request]
+jobs:
+  commits_check_job:
+    name: "Commits Check"
```

Comment:
Can we change the `name` of this job from "Commits Check" to "Validate presence of DCO information"?

Reply:
sure @grenade. Thank you for suggestion. will do that.